lib/Translation/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php

```diff
@@ -145,8 +145,8 @@ protected function addSettings(Document $document, AttributeSet $result): Docume
                     $targetDocument = Document::getById($translations[$targetLanguage]);
 
                     if ($targetDocument instanceof  Document\PageSnippet) {
-                        $targetData['title'][$targetLanguage] = $document->getTitle();
-                        $targetData['description'][$targetLanguage] = $document->getDescription();
+                        $targetData['title'][$targetLanguage] = $targetDocument->getTitle();
```

Comment:
please change the if check to `if ($targetDocument instanceof Document\Page) {` to fix the failing tests. thanks!